artemis/src/main/java/tech/pegasys/artemis/BeaconNode.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis;
+
+import com.google.common.eventbus.AsyncEventBus;
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import java.io.IOException;
+import java.util.concurrent.Executors;
+import net.consensys.cava.bytes.Bytes32;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.config.Configurator;
+import picocli.CommandLine;
+import tech.pegasys.artemis.data.RawRecord;
+import tech.pegasys.artemis.data.TimeSeriesRecord;
+import tech.pegasys.artemis.data.provider.CSVProvider;
+import tech.pegasys.artemis.datastructures.blocks.BeaconBlock;
+import tech.pegasys.artemis.datastructures.state.BeaconState;
+import tech.pegasys.artemis.networking.p2p.MockP2PNetwork;
+import tech.pegasys.artemis.networking.p2p.api.P2PNetwork;
+import tech.pegasys.artemis.services.ServiceController;
+import tech.pegasys.artemis.services.beaconchain.BeaconChainService;
+import tech.pegasys.artemis.services.chainstorage.ChainStorageService;
+import tech.pegasys.artemis.services.powchain.PowchainService;
+import tech.pegasys.artemis.util.cli.CommandLineArguments;
+import tech.pegasys.artemis.util.hashtree.HashTreeUtil;
+
+public class BeaconNode {
+  private static final Logger LOG = LogManager.getLogger(CSVProvider.class.getName());
```

Comment:
hmm I think this is supposed to be indented, did you run spotless? :)

Reply:
lemme check...i don't know how you all notice stuff like that.  seriously, makes me think there is something wrong with my brain

Reply:
ok so i ran spotless.....no change

Reply:
It's not our code! So it's easier to spot 😛